core/common/src/main/java/alluxio/master/PollingMasterInquireClient.java

```diff
@@ -88,7 +88,7 @@ private void pingMetaService(InetSocketAddress address)
       throws UnauthenticatedException, TTransportException {
     TTransport transport = TransportProvider.Factory.create().getClientTransport(address);
     TProtocol protocol =
-        ThriftUtils.createThriftProtocol(transport, Constants.META_MASTER_SERVICE_NAME);
+        ThriftUtils.createThriftProtocol(transport, Constants.META_MASTER_CLIENT_SERVICE_NAME);
```

Comment:
This will make it so that old clients can't talk to new masters. Updating the constant name is OK, but can we keep the original value of `"MetaMaster"`?